tensorflow/python/keras/layers/convolutional.py

```diff
@@ -77,6 +77,16 @@ class Conv(Layer):
       the dilation rate to use for dilated convolution.
       Currently, specifying any `dilation_rate` value != 1 is
       incompatible with specifying any `strides` value != 1.
+    groups: Integer, the number of channel groups controlling the connections
+      between inputs and outputs. Input channels and `filters` must both be
+      divisible by `groups`. For example,
```

Comment:
(for tensorflow/api-owners)

The following seems to be copy pasted from PyTorch [documentation](https://pytorch.org/docs/stable/nn.html#torch.nn.Conv2d) and should be rephrased.

Reply:
The structure is heavily inspired by PyTorch because I thought it describes the `groups` argument quite clearly.
@jsimsa Coul you elaborate on what part of the docstring is unclear, or should I just rephrase it so it doesn't resemble the PyTorch version as closely?

Reply:
You need to rephrase this.

Reply:
@jsimsa I updated the docstrings in 389fba4b3c9337ae2930b605f8cae3891c8b60d7

I hope this makes them clearer and easier to understand.